test/e2e/assets/kc-e2e-test-repo/packages/pkg.test.carvel.dev/pkg.test.carvel.dev.2.0.0.yml

```diff
@@ -8,8 +8,10 @@ spec:
   # In this new version of the package, we are including some overlays
   # which will increase the pod count for the simple app.
   version: 2.0.0
+  releasedAt: 2021-05-05T18:57:06Z
```

Comment:
Since this is for illustrative purposes, would it actually make sense to show this as released later than version 1.0.0?